help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -123,6 +143,15 @@ Navigate to your Advertiser ID → Audiences → Advertiser Audiences. If your a
 
 ![Amazon DSP audience creation validation](../../assets/catalog/advertising/amazon_ads_image_3.png)
 
+**For Amazon Marketing Cloud**
+
+In the left-hand schema browser, find your audience under Advertiser Uploaded → aep_audiences. You may then query your audience in the AMC SQL editor with the following clause:
+
+select count(user_id) from aep_audiences where audienceId = '1234567'
+
+![Amazon Marketing Cloud audience creation validation]()
```

Comment:
it looks like the image that you were going to link to is missing. the path to the image should be between the parentheses, like this:

` ![Amazon Marketing Cloud audience creation validation](path-to-image.png)`